src/lib/pdb/CopyPdbToArtifacts.ps1

```diff
@@ -0,0 +1,20 @@
+# BinSkim, one step in static analysis, requires PDB to analyze assemblies.
+# In most cases it can find them without extra configuration.
+# However for special cases, we need to copy the PDB to the same folder as the assemblies so that BinSkim can find them,,
+# and that is the purpose of this script.
+
+Param(
+    [string]$Configuration = "Debug"
+)
+
+$PathMappings = @{
+    'msalruntime.pdb' = 'Az.Accounts/lib/netstandard2.0'
+    'msalruntime_x86.pdb' = 'Az.Accounts/lib/netstandard2.0'
+}
+
+$ArtifactsPath = [System.IO.Path]::Combine($PSScriptRoot, "../../../artifacts", $Configuration)
```

Comment:
Could you use pure PowerSHell script? Such as `Resolve-Path`

Reply:
I favor .NET APIs because they perform better. Though in this case it matters little.